test/lib/ansible_test/_internal/util.py

```diff
@@ -694,6 +694,12 @@ def common_environment() -> dict[str, str]:
         'CFLAGS',
     )
 
+    # Allow additional environment variables to be passed to subprocesses.
+    # Use with caution, as additional environment variables may have unintended consequences.
+    # Variables are only exposed on the origin host and will not be relayed to delegated hosts.
+    pass_env = tuple(os.environ.get('ANSIBLE_TEST_PASS_ENV', '').split())
```

Comment:
Would it be better to use an `shlex.split()`?

Reply:
Spaces are not valid in environment variable names, so quoting support is unnecessary.